pages/common/subfinder.md

```diff
@@ -0,0 +1,24 @@
+# subfinder
+
+> SubFinder is a subdomain discovery tool that discovers valid subdomains for websites.
+> Designed as a passive framework to be useful for bug bounties and safe for penetration testing.
+
+- Find subdomains for example.com:
```

Comment:
Perhaps change to `for a specific domain`. It's implied that `example.com` is the example.